Admin/AdminHelper.php

```diff
@@ -284,13 +285,11 @@ public function getElementAccessPath($elementId, $entity)
                     $partReturnValue .= $dot.$id;
                     $dot = '.';
                     $id = '';
-                } else {
-                    $dot = '';
                 }
             }
 
-            if ($dot !== '.') {
-                throw new \Exception(sprintf('Could not get element id from %s Failing part: %s', $elementId, $subValue));
+            if (!empty($id)) {
```

Comment:
Because if you throw the exception on the else part, you stop looking for attributes matching.

For example, if your attribute is called more_part. The foreach first look at more and it goes to the 'else' part. So if you throw an exception there it will stop looking at 'parts'. Without the else it will look for more_parts in the next iteration